src/containers/SellerContainer.jsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import moment from 'moment';
+import { addProduct } from '../states/features/product/productSlice';
+
+const SellerContainer = () => {
+  const dispatch = useDispatch();
+  const [product, setProduct] = useState(null);
```

Comment:
Please include in your redux, product state. Please maintain uniformity. Check how others have done it, and do the same.